sampleapi/api_ok.go

```diff
@@ -1,10 +1,28 @@
 package sampleapi
 
+import (
```

Comment:
func main() {
	var wg sync.WaitGroup
	wg.Add(2)

	go func() {
		fmt.Println("Hello World 1")
		wg.Done()
	}()

	go func() {
		fmt.Println("Hello World 2")
		wg.Done()
	}()

	wg.Wait()
}

// This code looks good! It is concise, easy to read, and uses Go's built-in synchronization primitives. The only suggestion I might have is to use consistent formatting, such as using the same number of spaces for indentation.